packages/langchain/src/orchestration/client.ts

```diff
@@ -31,6 +38,15 @@ export class OrchestrationClient extends BaseChatModel<
     public deploymentConfig?: ResourceGroupConfig,
     public destination?: HttpDestinationOrFetchOptions
   ) {
+    // Avoid retry if the error is due to input filtering
+    const { onFailedAttempt } = langchainOptions;
+    langchainOptions.onFailedAttempt = error => {
```

Comment:
To better understand how this works -> https://github.com/sindresorhus/p-retry?tab=readme-ov-file#onfailedattempterror